src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,42 @@
      * Андрей - 600
      * София - 900</p>
      */
+
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate finishDate = LocalDate.parse(dateTo, FORMATTER);
+
+        if (startDate.isAfter(finishDate)) {
+            throw new IllegalDateParametersException("Wrong parameters");
+        }
+        StringBuilder res = new StringBuilder();
```

Comment:
let's give more informative name for this variable

Reply:
Done as advised above.